manifests/README.md

```diff
@@ -87,6 +87,7 @@ For deeper documentation to deploy CFCR go [here](https://github.com/cloudfoundr
 | [`ops-files/enable-denyescalatingexec.yml`](ops-files/enable-denyescalatingexec.yml) | Enables the DenyEscalatingExec admission plugin. | This ops-file is recommended for most clusters. | - |
 | [`ops-files/enable-securitycontextdeny.yml`](ops-files/enable-securitycontextdeny.yml) | Enables the SecurityContextDeny admission plugin. | This ops-file is recommended for most clusters. | - |
 | [`ops-files/enable-podsecuritypolicy.yml`](ops-files/enable-podsecuritypolicy.yml) | Enables the PodSecurityPolicy admission plugin. | Please ensure that you have applied an appropriate policy before enabling this plugin.  Failure to do so will result in failure of your workloads. |
+| [`ops-files/change-audit-log.yml`](ops-files/change-audit-log.yml) | Allow to change the parameters of kubernetes audit log. see https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#log-backend to have more details  | - |
```

Comment:
Just being somewhat nitpicky here: the file might be better called `change-audit-log-flags.yml` just to differentiate between changing the flags and changing the audit log policy (which isn't currently configurable, but likely will be soon).

Also, the wording on the quote is a bit odd. Perhaps something more like:

> Changes the kube-apiserver's audit-log flags. See [here](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#log-backend) for more details.

Reply:
@tvs It is one